pkgs/applications/misc/tuckr/fix-build.patch

```diff
@@ -0,0 +1,49 @@
+diff --git i/Cargo.lock w/Cargo.lock
+index 5f5dd43..d7c8370 100644
+--- i/Cargo.lock
++++ w/Cargo.lock
```

Comment:
1. How was this generated?
2. Will this be upstreamed someday?

Reply:
1. By me manually
2. It will be fixed for the next release